### DISCUSSION TITLE
Add Linksys EA6900 support

### DISCUSSION THREAD

**Discussion by ansilent** [SWRT-dev/swrt-gpl#28] [Ideas]
Please add Linksys EA6900 support. This same model as EA6700 but different WAN/ LAN port numeration.

**Comment by paldier** [comment]
I don‘t have EA6900.

**Reply by ansilent** [reply]
I have EA6900 and I can help with tests.
I installed your firmware for RT-AC68U. With this firmware WAN/ LAN port numeration is ok, but don't work USB 2.0 port  and don't work led indicator.
Then I installed your firmware for EA6700. With this firmware both USB ports is work, led indicator work too. But WAN/ LAN numeration is not good:
EA6900 WAN port defined as LAN1 in web interface
EA6900 LAN1 -----> LAN2
EA6900 LAN2 -----> LAN3
EA6900 LAN3 -----> LAN4
EA6900 LAN4 -----> WAN
Need only change it the same as for RT-AC68U

**Reply by paldier** [reply]
Xwrt-Vortex firmware already supports this model.
